src/mame/drivers/coco12.cpp

```diff
@@ -730,6 +748,7 @@ COMP( 19??,  coco2bh,   coco,   0,      coco2bh, coco,    coco12_state, empty_in
 COMP( 1983,  cp400,     coco,   0,      cp400,   coco,    coco12_state, empty_init, "Prológica",                    "CP400",                               0 )
 COMP( 1985,  cp400c2,   coco,   0,      cp400,   cp400c2, coco12_state, empty_init, "Prológica",                    "CP400 Color II",                      0 )
 COMP( 1983,  lzcolor64, coco,   0,      coco,    coco,    coco12_state, empty_init, "Novo Tempo / LZ Equipamentos", "Color64",                             0 )
-COMP( 1983,  cd6809,    coco,   0,      cd6809,  coco,    coco12_state, empty_init, "Codimex",                      "CD-6809",              0 )
+COMP( 1983,  cd6809,    coco,   0,      cd6809,  coco,    coco12_state, empty_init, "Codimex",                      "CD-6809",                             0 )
 COMP( 1984,  mx1600,    coco,   0,      coco,    coco,    coco12_state, empty_init, "Dynacom",                      "MX-1600",                             0 )
 COMP( 1986,  t4426,     coco,   0,      t4426,   coco,    coco12_state, empty_init, "Terco AB",                     "Terco 4426 CNC Programming station",  0 )
+COMP( 1987,  ms1600,    coco,   0,      ms1600,  coco,    coco12_state, empty_init, "Instituto Latinoamericano de la Comunicación Educativa / Secretaría de Educación Pública","Micro-SEP 1600",0 )
```

Comment:
Please put this in the same order here as you have in the ROM declarations.  Also, consider that the manufacturer needs to be displayed in the internal UI.  This is going to be way too long.  Please use the official abbreviations “ILCE” and “SEP”, and let an external DAT file provide the full form.  (I verified that the abbreviations “ILCE” and “SEP” are used in official press material from these organisations.)